docs/welcome/proposing-content.md

```diff
@@ -0,0 +1,35 @@
+# Proposing a Project or Course
+
+Here at DataCamp,
+we recognize that our community knows a lot more than we ever will,
+so we are always open to proposals for new courses or projects.
+If you would like to propose something:
+
+1. Have a look at our current projects and courses to see if there's any overlap with existing content.
+
+2. Skim [our course design description](../courses/design/)
+   and [our project creation description](../projects/projects-process.html)
+   and decide which you think would be a better fit for what you'd like to teach.
+
+3. If you want to create a course,
+   write a single-paragraph course description like [the examples given here](../courses/design/overview-description.html),
+   along with one example exercise similar to [these examples](../courses/design/exercises-examples.html).
+   (We strongly prefer plain text formatted as Markdown,
+   but will accept PDFs as well if you need to include graphics to showcase your ideas.)
+
+4. If you want to create a project,
+   create a Jupyter Notebook that has a short description designed to entice learners
+   and a couple of short code fragments to show what learners will do,
+   similar to [the examples given here](../projects/projects-process.html#projects-writing-specification).
+
+5. Email your idea to FIXME.
```

Comment:
we should probably set up teach@datacamp.com (better aligns with our branding compared to create@datacamp.com) and funnel everything to that email (same with applications from the /create page). Worth a quick discussion in the curriculum team meeting tomorrow